apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -153,58 +151,40 @@ export default {
       }
     },
 
-    $_ownerAndResharers () {
-      const ownerArray = this.$_shareOwnerAsCollaborator ? [this.$_shareOwnerAsCollaborator] : []
-      return [
-        ...ownerArray,
-        ...this.$_resharersAsCollaborators
-      ]
-    },
-    $_shareOwnerAsCollaborator () {
+    $_ownerShare () {
       if (!this.$_allIncomingShares.length) {
         return null
       }
 
-      const firstShare = this.$_allIncomingShares[0]
-      return {
-        ...firstShare,
-        name: firstShare.info.uid_file_owner,
-        displayName: firstShare.info.displayname_file_owner,
-        key: 'owner-' + firstShare.info.id,
-        info: {
-          path: firstShare.info.path,
-          // set to user share for displaying user
-          share_type: '' + shareTypes.user, // most code requires string..
-          share_with_additional_info: firstShare.info.additional_info_file_owner || []
-        },
-        role: this.ownerRole
-      }
-    },
-
-    $_resharersAsCollaborators () {
-      const resharers = new Map()
-      const owner = this.$_shareOwnerAsCollaborator
+      const ownerShare = this.$_allIncomingShares[0]
 
+      let resharers = new Map()
       this.$_allIncomingShares.forEach(share => {
-        if (share.info.uid_owner !== owner.name) {
+        if (share.info.uid_owner !== ownerShare.name) {
           resharers.set(share.info.uid_owner, {
-            ...share,
             name: share.info.uid_owner,
-            displayName: share.info.displayname_owner,
-            role: this.resharerRole,
-            key: 'resharer-' + share.info.id,
-            info: {
-              path: share.info.path,
-              // set to user share for displaying user
-              share_type: '' + shareTypes.user, // most code requires string..
-              share_with_additional_info: share.info.additional_info_owner || []
-            }
+            displayName: share.info.displayname_owner
           })
         }
       })
 
       // make them unique then sort
-      return Array.from(resharers.values()).sort(this.$_collaboratorsComparator.bind(this))
+      resharers = Array.from(resharers.values()).sort(this.$_collaboratorsComparator.bind(this))
+
+      return {
+        ...ownerShare,
+        name: ownerShare.info.uid_file_owner,
+        displayName: ownerShare.info.displayname_file_owner,
+        key: 'owner-' + ownerShare.info.id,
+        info: {
+          path: ownerShare.info.path,
+          // set to user share for displaying user
+          share_type: '' + shareTypes.user, // most code requires string..
```

Comment:
We recently spent some time into providing integers where possible, but not in `collaborator.info.share_type`, but in `collaborator.shareType`.

Reply:
I killed all usage of "info" now as I grew tired of it, so this is gone as well